### PR TITLE
chore(gradle): update to Java 8

### PR DESCRIPTION
**Pull request by ariporad**
Ready for Review

### CODE REVIEW

File: build.common.gradle [hunk 1]
```diff
@@ -38,6 +38,12 @@ android {
         minSdkVersion 19
         targetSdkVersion 19
 
+        // ADDED BY @ARIPORAD
+        jackOptions {
+            enabled true
+        }
+        // END ADDED BY @ARIPORAD
```
Comment (strugee) [inline build.common.gradle]: Drive-by comment: why don't you just use blame?

Comment (ariporad) [inline build.common.gradle]: @strugee: Because I'm lazy. If I'm updating the FTC SDK, I just want to be able to do a CMD-F for "ADDED BY" and figure out the bits I need to merge manually vs. just pasting over. I don't actually care who added it.

**Comment by ariporad** [comment]
@NoahRoseLedesma: Thoughts?

**Comment by NoahRoseLedesma** [comment]
Looks good, just want to make sure it doesn't break anything on other systems. Ill check that out in a bit.

**Comment by ariporad** [comment]
Thx

File: build.common.gradle (hunk 1 above)
Comment (strugee) [inline build.common.gradle]: @ariporad not sure I understand 100% of what you're saying but it sounds like you could do the same thing serching for `<<<<<<`, `======` and `>>>>>>>`? Whatever tho :P

Comment (ariporad) [inline build.common.gradle]: @strugee: "Shut up let me have my fun"
